src/sentry/integrations/slack/utils.py

```diff
@@ -162,6 +162,13 @@ def build_action_text(group, identity, action):
     )
 
 
+def build_rule_url(rule, group, project):
+    org_slug = group.organization.slug
+    project_slug = project.slug
+    rule_url = u"/settings/{}/projects/{}/alerts/rules/{}".format(org_slug, project_slug, rule.id)
```

Comment:
@KKaempen  For whatever reason, Sentry has a trailing `/` at the end of every URL. Front end routes work without it but API routes don't so we just always use them for our URLs.

Reply:
Ah OK, I'll fix that. Looks like I might need a rebase, so I'll do that while I'm at it.